assemble/conf/create-jshell.sh

```diff
@@ -0,0 +1,134 @@
+#! /usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+function addLicenseHeader(){
+  export header="$1"
+  export jPath="$2"
+  
+  # Does an auto-generated JShell config file exists?
+  if [[ -e "$jPath" ]]; then
+    printf "%s\n" "/*" >> "$jPath"
+    
+    # Load in license header
+    while IFS= read -r line; do
+      echo "$line" >> "$jPath"
+    done < "$header"
+    
+    printf "%s\n" "*/" >> "$jPath" 
+    echo " " >> "$jPath"
+  else
+    echo "Cannot add license header in $jPath"
+    echo "Please ensure jshell-init.jsh exists"
+    exit 1
+  fi
```

Comment:
The generated file doesn't hold any meaningful/copyrightable content. It's just generated import statements, so it's not strictly necessary to add the license header. The rat check error from the missing license header can be avoided by generating the file in the build tree, rather than the source tree. For maven, that means generating it in `assemble/target/` directory rather than `assemble/conf/`

That should simplify things a bit.

Reply:
@ctubbsii Thank you for reviewing my code submission. I'll make the proposed updates and test out the code before pushing a revised copy later.